runtime/src/bank.rs

```diff
@@ -5159,10 +5175,51 @@ impl Bank {
         );
         check_time.stop();
 
+        let sanitized_output = self.load_and_execute_sanitized_transactions(
+            sanitized_txs,
+            &mut check_results,
+            &mut error_counters,
+            enable_cpi_recording,
+            enable_log_recording,
+            enable_return_data_recording,
+            timings,
+            account_overrides,
+            log_messages_bytes_limit,
+            &mut check_time,
+        );
+        LoadAndExecuteTransactionsOutput {
+            loaded_transactions: sanitized_output.loaded_transactions,
+            execution_results: sanitized_output.execution_results,
+            retryable_transaction_indexes,
+            executed_transactions_count: sanitized_output.executed_transactions_count,
+            executed_non_vote_transactions_count: sanitized_output
+                .executed_non_vote_transactions_count,
+            executed_with_successful_result_count: sanitized_output
+                .executed_with_successful_result_count,
+            signature_count: sanitized_output.signature_count,
+            error_counters,
+        }
+    }
+
+    #[allow(clippy::type_complexity)]
```

Comment:
don't need this marker anymore since the return type was made into a proper struct